guides/walkthroughs/how-to-complete-stellar-toml.md

```diff
@@ -0,0 +1,250 @@
+How and Why to Complete Your Stellar.toml
+=========================================
+
+>*If you are interested in issuing a token on the Stellar network, but haven't yet, start by consulting the step-by-step instructions for custom asset creation *[*here*](https://www.stellar.org/developers/guides/walkthroughs/custom-assets.html)*.*
```

Comment:
For references to other pages within the docs, I believe we use relative, not absolute links. 

For example, here I'd use `./custom-assets.md`